packages/shared/src/write-archive/dom-snapshot.ts

```diff
@@ -162,9 +162,14 @@ export class DOMSnapshot {
 
   private mapCssUrls(cssText: string, sourceMap: Map<string, string>) {
     return cssText.replace(CSS_URL_REGEX, (match, fullUrl) => {
-      let cssUrl = match;
-      if (sourceMap.has(fullUrl)) {
-        cssUrl = match.replace(fullUrl, sourceMap.get(fullUrl));
+      const normalizedUrl = fullUrl.replace(/\\/g, '/').replace(/([^:])\/\/+/, '$1/');
+      console.log('normalizedUrl', normalizedUrl);
```

Comment:
There are a few `console.log`s that need to be removed

Reply:
Ahhh man, I thought I had gotten them all - my bad!